frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-resource.tsx

```diff
@@ -28,29 +30,45 @@ export const VMTemplateResourceSummary: React.FC<VMTemplateResourceSummaryProps>
   const id = getBasicID(template);
   const base = getVmTemplate(template);
 
-  const description = getDescription(template) || DASH;
+  const description = getDescription(template);
+  const os = getOperatingSystemName(template) || getOperatingSystem(template);
+  const workloadProfile = getWorkloadProfile(template);
 
   return (
     <ResourceSummary resource={template}>
-      <dt>Description</dt>
-      <dd id={prefixedID(id, 'description')} className="kubevirt-vm-resource-summary__description">
+      <VMDetailsItem
+        title="Description"
+        idValue={prefixedID(id, 'description')}
+        valueClassName="kubevirt-vm-resource-summary__description"
+        isNotAvail={!description}
+      >
         <EditButton
           canEdit={canUpdateTemplate}
           onClick={() => vmDescriptionModal({ vmLikeEntity: template })}
         >
           {description}
         </EditButton>
-      </dd>
-      <dt>Operating System</dt>
-      <dd id={prefixedID(id, 'os')}>
-        {getOperatingSystemName(template) || getOperatingSystem(template) || DASH}
-      </dd>
-      <dt>Workload Profile</dt>
-      <dd id={prefixedID(id, 'workload-profile')}>{getWorkloadProfile(template) || DASH}</dd>
-      <dt>Base Template</dt>
-      <dd id={prefixedID(id, 'base-template')}>
-        {base ? <VMTemplateLink template={base} /> : DASH}
-      </dd>
+      </VMDetailsItem>
+
+      <VMDetailsItem title="Operating System" idValue={prefixedID(id, 'os')} isNotAvail={!os}>
+        {os}
+      </VMDetailsItem>
+
+      <VMDetailsItem
+        title="Workload Profile"
+        idValue={prefixedID(id, 'workload-profile')}
+        isNotAvail={!workloadProfile}
+      >
+        {workloadProfile}
+      </VMDetailsItem>
+
+      <VMDetailsItem
+        title="Base Template"
+        idValue={prefixedID(id, 'base-template')}
+        isNotAvail={!template}
```

Comment:
this should be a bit safer, right ?
```suggestion
        isNotAvail={!base}
```

Reply:
good catch